app/hotels/src/map/MapView.js

```diff
@@ -34,10 +34,7 @@ type MarkerPressEvent = {|
   },
 |};
 
-type LatLng = {|
-  latitude: number,
-  longitude: number,
-|};
+const NO_OF_MARKERS_IN_REGION = 15;
```

Comment:
It still looks very crowded. What about fetching just the first 15? Airbnb has a similar approach (+lazy fetching of additional hotels - we don't need this now).